README.md

```diff
@@ -49,7 +86,7 @@ icestore.registerStore('todos', todos);
 export default icestore;
 ```
 
-在 View 中使用：
+* In view component, import store instance, get store config (including state and actions) by using `useStore` hook, then trigger actions from `useEffect` hook or event callbacks, finally bind state to view template.
```

Comment:
In the view component, you can get the store config (including state and actions) by using the `useStore` hook after importing the store instance. After that, you can trigger actions through event callbacks or by using the `useEffect` hook, which binds the state to the view template.